webapp/channels/src/components/sidebar/invite_members_button.tsx

```diff
@@ -52,11 +38,11 @@ const InviteMembersButton = (props: Props): JSX.Element | null => {
 
     let buttonClass = 'SidebarChannelNavigator__inviteMembersLhsButton';
 
-    if (!props.touchedInviteMembersButton && Number(totalUserCount) <= Constants.USER_LIMIT) {
+    if (!props.touchedInviteMembersButton) {
```

Comment:
Maybe we should remove the feature for everyone? Rather than enabling it for bigger servers?